source/Engine.cpp

```diff
@@ -790,8 +790,20 @@ void Engine::Click(const Point &from, const Point &to, bool hasShift)
 	doClickNextStep = true;
 	this->hasShift = hasShift;
 	isRightClick = false;
-	clickPoint = from;
-	clickBox = Rectangle::WithCorners(from / zoom + center, to / zoom + center);
+	
+	// Determine if the left-click was within the radar display.
+	const Point radarCenter = GameData::Interfaces().Get("targets")->GetPoint("radar");
+	static const double radarRadius = GameData::Interfaces().Get("targets")->GetSize("radar").Y() * .5;
```

Comment:
As with other UI values, it's probably worthwhile to have this be non-static in case future game changes make it possible to reconfigure the heads-up-display interface while in-game.

Reply:
Addressed